plugin/trino-iceberg/src/test/java/io/trino/plugin/iceberg/TestIcebergConnectorSmokeTest.java

```diff
@@ -35,7 +36,11 @@
     protected QueryRunner createQueryRunner()
             throws Exception
     {
-        return createIcebergQueryRunner(ImmutableMap.of(), ImmutableMap.of(), REQUIRED_TPCH_TABLES);
+        return createIcebergQueryRunner(
+                ImmutableMap.of(),
+                ImmutableMap.of(),
+                Optional.empty(),
+                REQUIRED_TPCH_TABLES);
```

Comment:
that would be reverted if you accept my other suggestion

Reply:
Applied